packages/appstore/src/components/containers/trading-app-card.scss

```diff
@@ -75,5 +75,8 @@
 
     &__title {
         user-select: none;
+        &-bold {
+            font-weight: bold;
+        }
```

Comment:
can't we handle this in `Text` component?

Reply:
This classname is not used, I handled it in the Text component. Removing it to not cause any confussion.